tcs-api/src/main/java/com/transformuk/hee/tis/tcs/api/dto/ProgrammeMembershipSummaryDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.transformuk.hee.tis.tcs.api.dto;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+import lombok.Data;
+
+/**
+ * A DTO for the ProgrammeMembershipSummary entity.
+ */
+@Data
+public class ProgrammeMembershipSummaryDTO implements Serializable {
+  private String programmeMembershipId;
```

Comment:
Not sure what field is meant here: is it the uuid? If so, it should probably be called 'uuid' (the 'id' field in the ProgrammeMembership entity is deprecated).